### PR TITLE
Add explanation describing chain splits

### PR DESCRIPTION
**Pull request by wbnns**
Since the [alert](https://bitcoin.org/en/alert/2017-07-12-potential-split) went live last night, I've read a lot of feedback from people who are confused about what a chain split is and are asking others to ELI5 (explain like I'm 5).

This PR adds a brief, high level overview of what a chain split is to the beginning of the post. Unless there is any strong objection that anything I've said is fundamentally incorrect in describing a chain split, I will merge this today at 18:00 UTC.  Also, if anyone would like to suggest any improvements or additional copy, please feel free to submit a PR to my branch before then (`wbinns-split-updates-0713171`), or open a new PR or Issue later today.

### CODE REVIEW

File: _alerts/2017-07-12-potential-split.md [hunk 1]
```diff
@@ -12,20 +12,39 @@ bannerclass: "warning"
...
-
-Once the situation is resolved, confirmation scores will either
-automatically return to their normal reliability or there will be two
-(or more) competing versions of Bitcoin.  In the former case, you may
-return to using Bitcoin normally; in the later case, you will need to
-take extra steps in order to begin safely receiving bitcoins again.
+## Chain split
+
+Since Bitcoin's inception, the predominant software that has been used across
+the network has been a program called Bitcoin Core. During this time, Bitcoin has
```
Comment (harding) [inline _alerts/2017-07-12-potential-split.md]: "called Bitcoin Core".  Not true; it used to be called just "Bitcoin".  The name was changed in version 0.8 IIRC.

File: _alerts/2017-07-12-potential-split.md [hunk 2]
```diff
@@ -12,20 +12,39 @@ bannerclass: "warning"
...
+grown immensely. The network has facilitated billions of transactions for
```
Comment (harding) [inline _alerts/2017-07-12-potential-split.md]: "billions of transactions" Not true; its 238,962,626 transactions as of the most recent block, 00000000000000000176c021933ac22cfff38b4ac9ca4536e6aa7e8554dd8283

File: _alerts/2017-07-12-potential-split.md [hunk 3]
```diff
@@ -12,20 +12,39 @@ bannerclass: "warning"
...
+millions of users. As a result, different groups of people (developers,
+investors, entrepreneurs, etc.), have debated on the best ways Bitcoin can be
+optimized to allow it to exponentially scale to even more users and
+transactions. In recent months alternative software has been released that
+represent some of the interests of the aforementioned groups of people - software
+that is scheduled to go live toward the end of July.
+
+The Bitcoin block chain, which is a record of all Bitcoin transactions to date,
+relies on a network of thousands of Bitcoin nodes running Bitcoin software that
+help propagate transactions. On {{start}}, the Bitcoin block chain may experience
```
Comment (harding) [inline _alerts/2017-07-12-potential-split.md]: "... help propagate transactions"  This is completely unrelated to the issue at hand.

File: _alerts/2017-07-12-potential-split.md [hunk 4]
```diff
@@ -12,20 +12,39 @@ bannerclass: "warning"
...
+what is known as a chain split. This is when a portion of Bitcoin's nodes
+run software that another portion of nodes are not fully compatible with. As a
+result, some nodes may propagate confirmed transactions that other nodes may
+not accept or recognize due to alternative software. This may result in
```
Comment (harding) [inline _alerts/2017-07-12-potential-split.md]: "some nodes may propagate confirmed transactions that other nodes may not accept or recognize" this is a poor description of a problem that's less significant than the main problem.  The main problem is that you will think you have bitcoins when you don't, and so you'll do things like trade real money, products, or services for bitcoins that have no value.

**Comment by wbnns** [comment]
@harding Hey, you're right, I did mention that we need an introductory paragraph yesterday before we went live, and since then there has been a lot of user feedback on places like Hacker News or Reddit where people are saying that they are confused about what is going on.

Thanks for the feedback on the other points though, I'm updating that now.

File: _alerts/2017-07-12-potential-split.md (hunk 2 above)
Comment (wbnns) [inline _alerts/2017-07-12-potential-split.md]: My bad, I was looking at the wrong chart, total value of transactions - thanks for catching. Fixed.

File: _alerts/2017-07-12-potential-split.md (hunk 1 above)
Comment (wbnns) [inline _alerts/2017-07-12-potential-split.md]: I went ahead and just struck this part so we don't have to split hairs, I was just referring to it by its current name so people could reference if they'd like.

File: _alerts/2017-07-12-potential-split.md (hunk 3 above)
Comment (wbnns) [inline _alerts/2017-07-12-potential-split.md]: Struck `help propagate transactions` from the copy.

File: _alerts/2017-07-12-potential-split.md (hunk 4 above)
Comment (wbnns) [inline _alerts/2017-07-12-potential-split.md]: The transactional risk ("main problem" that you mentioned) is already described in other places, so I think the additional descriptor in this PR should be left in to help explain to people who have voiced confusion over the previous version.

**Comment by wbnns** [comment]
Ok, updated. I think the chain split info should still go at the beginning. People are confused, for example, here is the most upvoted comment on Hacker News (this post is also one of the top trending items on Hacker News, now):

image

@harding It's important with this kind of update that we be aware of user feedback and try to make sure this post is as helpful as possible to users. The original post did not adequately do that as you can see from plenty of comments from dozens of people.

**Comment by harding** [comment]
@wbnns given that you plan to merge over my NACK, I leave responsibility for this document in your hands both now and after August 1st.  I've found it very difficult to work with you on this issue from the start and I think it's probably a waste of time for both of us to continue arguing.

**Comment by wbnns** [comment]
@harding Ok, no problem - as mentioned, you're welcome (as is anyone else) to submit a PR to my branch or open a new separate issue or PR. I'm also up for working on this to make any further revisions (in addition to the ones I just made based on your feedback).

**Comment by achow101** [comment]
If you're going to include information about alternative clients causing a chain split, then you will need to mention BIP 91. I think just a brief mention that there is another proposal (BIP 91) that has the potential to activate and cause a chain split prior to august 1st, but it requires 80% hash rate to activate and there are currently no miners signalling for it. However 80%+ of the hash rate has verbally agreed to activate that proposal some time this month.

Could you also please wait for other people to review any changes to the page before merging? I would like to review them and I will try to review them as quickly as possibly.

File: _alerts/2017-07-12-potential-split.md [hunk 5]
```diff
@@ -12,20 +12,37 @@ bannerclass: "warning"
...
+split. This is when a portion of Bitcoin's nodes run software that another
+portion of nodes are not fully compatible with. As a result, some nodes may
+propagate confirmed transactions that other nodes may not accept or recognize
+due to alternative software. This may result in unreliable [confirmation
+scoring][] for an unknown length of time.  This means that any bitcoins you
+receive during this period could disappear from your wallet or be a type of
+bitcoin that other people will not accept as payment, until the situation is
+resolved.
+
+Once the situation is resolved, confirmation scoring will either automatically
```
Comment (achow101) [inline _alerts/2017-07-12-potential-split.md]: confirmation scoring should be linked.

File: _alerts/2017-07-12-potential-split.md [hunk 6]
```diff
@@ -12,20 +12,37 @@ bannerclass: "warning"
...
-automatically return to their normal reliability or there will be two
-(or more) competing versions of Bitcoin.  In the former case, you may
-return to using Bitcoin normally; in the later case, you will need to
-take extra steps in order to begin safely receiving bitcoins again.
+## Chain split
+
+Since Bitcoin's inception, its network has facilitated hundreds of millions of
+transactions. As a result, different groups of people (developers, investors,
+entrepreneurs, etc.), have debated on the best ways Bitcoin can be optimized to
+allow it to exponentially scale to even further. In recent months alternative
```
Comment (achow101) [inline _alerts/2017-07-12-potential-split.md]: There should be a comma between months and alternative.

Comment (wbnns) [inline _alerts/2017-07-12-potential-split.md]: @achow101 Thanks, fixed. 👍

File: _alerts/2017-07-12-potential-split.md (hunk 5 above)
Comment (wbnns) [inline _alerts/2017-07-12-potential-split.md]: I linked to it in the second paragraph, the first time it was mentioned:

image

**Comment by achow101** [comment]
ACK 3fbea0cf893080cc850bcffc8e291a936d234870

**Comment by wbnns** [comment]
@achow101 Thanks for reviewing. With regard to adding a note about BIP 91, could you please open a separate PR with the content you have in mind once this gets merged?

The reason I think it is important to get the current PR live soon is because as already mentioned, a lot of people are confused. In addition to people asking questions on Hacker News, Reddit and social media, the [International Business Times](http://www.ibtimes.co.uk/warning-bitcoin-network-disruption-imminent-cryptocurrency-splits-1630247) just went to print with an article mentioning that this is a statement being made to everyone by Bitcoin Core developers:

image

...which this is not (fixed in `f47370a`). The Associated Press also reached out to clarify if this was a statement from Core developers.

Once tests pass I'm going to merge this, and if anyone would like any subsequent changes, please open an issue or PR. I will be standing by to help work together so we can make sure any subsequent changes get brought in so we can make this as good and comprehensive as possible for everyone who reads it.

**Comment by achow101** [comment]
@wbnns Ok. I will be making two PRs in the near future, one with BIP 91 information and the other with some general instructions on choosing a chain to use and splitting coins.

**Comment by wbnns** [comment]
@achow101 Ok, thanks for the help.
